app/lib/system/error_reporting.rb

```diff
@@ -23,6 +23,10 @@ def report_error(exception, logger: Rails.logger, **parameters)
       end
     end
 
+    def report_sidekiq_error(exception, _context, config)
+      report_error(exception, logger: config.logger)
+    end
```

Comment:
Bugsnag provides its own builtin method to capture Sidekiq errors and report them. Regretably, Sidekiq 7 introduced some changes on the error reporting and the `bugsnag` gem haven't updated yet to adapt to them. So we need to bypass the Bugsnag report method and use our own.

Reply:
Add a TODO / create a JIRA to keep track of this?

Reply:
I created a Jira for bugsnag and some other gems that should be updated as well: https://issues.redhat.com/browse/THREESCALE-10207